src/picomsg/src/dummy_oper_db_client/dummy_oper_db_client.cpp

```diff
@@ -109,6 +143,65 @@ bool DummyOperDBObject::do_u64_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_
 	return true;
 }
 
+bool DummyOperDBObject::do_u16_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint16_t new_num) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d from %s to %hu",
+					field_id, iter->second.c_str(), new_num);
+	}
+	m_fields.emplace(field_id, std::to_string(new_num));
+	return true;
+}
+
+bool DummyOperDBObject::do_u8_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint8_t new_num) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d from %s to %hu",
+					field_id, iter->second.c_str(), new_num);
+	}
+	m_fields.emplace(field_id, std::to_string(new_num));
+	return true;
+}
+
+bool DummyOperDBObject::do_bool_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, uint8_t new_val) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d from %s to %d",
+					field_id, iter->second.c_str(), new_val);
+	}
+	m_fields.emplace(field_id, std::to_string((int)new_val));
+	return true;
+}
+
+bool DummyOperDBObject::do_string_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, const char *new_data) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d from %s to %s",
+					field_id, iter->second.c_str(), new_data);
+	}
+	m_fields.emplace(field_id, new_data);
+	return true;
+}
+
+bool DummyOperDBObject::do_union_set(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id, struct corm_union *str_val) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d from %s to %s",
+					field_id, iter->second.c_str(), str_val->str);
+	}
+	m_fields.emplace(field_id, str_val->str);
+	return true;
+}
+
+bool DummyOperDBObject::do_timeticks_set_curr_timestamp(const enum LIBOPER_DB_CLIENT_FIELD_IDS field_id) {
+	const auto iter = m_fields.find(field_id);
+	if (m_fields.end() != iter) {
+		PICOMSG_LOG(PICOMSG_LOG_INFO, "overwriting field ID %d timestamp",
+					field_id);
+	}
+	return true;
```

Comment:
The point of this module is to log writes to oper, so not writing anything here misses the point.
Instead add the current time as the value of the field_id